Source/ASTextNode2.mm

```diff
@@ -360,10 +360,10 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
   [self prepareAttributedString:mutableText];
   
   return @{
-           @"container": copiedContainer,
-           @"text": mutableText,
-           @"bgColor": self.backgroundColor
-           };
+    @"container": copiedContainer,
+    @"text": mutableText,
+    @"bgColor": self.backgroundColor ?: [NSNull null]
```

Comment:
@Adlai-Holler Let me know if we should do more like:

```
if (self.backgroundColor) {
  return @{
   @"container": copiedContainer,
    @"text": mutableText,
    @"bgColor": self.backgroundColor
  }
}

  return @{
   @"container": copiedContainer,
    @"text": mutableText
  }
```

And remove the NSNull check in the display method.